texlive-scripts/fmtutil.pl

```diff
@@ -128,6 +130,7 @@ BEGIN
   "no-engine-subdir",
   "no-error-if-no-engine=s",
   "no-error-if-no-format",
+  "no-fork",
```

Comment:
Add a command  line argument to disable forking even in presence of ForkManager module.